### ISSUE TITLE
forEachKey is not a function Promise

### ISSUE THREAD

**Issue by Mar8x** [coderaiser/cloudcmd#311]
Hi!

I got the following issue:

```
ubuntu@ip-10-10-10-203:~$ cloudcmd --config ~/.cloudcmd.json
forEachKey is not a function Promise {
  <rejected> TypeError: forEachKey is not a function
      at readConfig (file:///usr/local/lib/node_modules/cloudcmd/bin/cloudcmd.mjs:280:5)
      at async main (file:///usr/local/lib/node_modules/cloudcmd/bin/cloudcmd.mjs:190:5)
}
```
seems like the `forEachKey` does not return a resolve/reject. 
https://github.com/coderaiser/for-each-key/blob/master/lib/for-each-key.js

Note: v14.9.3 does work as expected.

* **Version** (`cloudcmd -v`): 
v15.0.2
* **Node Version** `node -v`: 
v15.5.1
`for-each-key@2.0.0`
`simport@1.0.4` 
* **OS** (`uname -a` on Linux): 
 Linux ip-10-10-10-203 4.4.0-1118-aws #132-Ubuntu SMP Fri Nov 13 18:22:45 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
* **Browser name/version**: 

* **Used Command Line Parameters**: 
`--config`
* **Changed Config**: 
```
{
    "editor": "edward"
}
```

**Comment by coderaiser** [comment]
Thanks a lot :), fixed with 394bf46e. Landed in [v15.0.3](https://github.com/coderaiser/cloudcmd/releases/tag/v15.0.3) 🎉 .

Is it works for you?

**Comment by Mar8x** [comment]
Thanks @coderaiser, worked like a charm :-)
